spec/models/aggregated_cash_outgoings_spec.rb

```diff
@@ -0,0 +1,478 @@
+require 'rails_helper'
+
+RSpec.describe AggregatedCashOutgoings, type: :model do
+  let(:aco) { described_class.new }
+  let(:application) { create :legal_aid_application }
+  let(:categories) { %i[rent_or_mortgage maintenance_out] }
+  let!(:rent_or_mortgage) { create :transaction_type, :rent_or_mortgage }
+  let!(:maintenance_out) { create :transaction_type, :maintenance_out }
+  let!(:child_care) { create :transaction_type, :child_care }
```

Comment:
:child_care is unused, it can be removed or the intention may have been to add it to the 'additional_valid_params' method